btcrecover/btcrseed.py

```diff
@@ -1338,7 +1349,7 @@ def _addresses_to_hash160s(addresses):
                     if c.isalpha() and \
                        c.isupper() != bool(ord(checksum[nibble // 2]) & (0b1000 if nibble&1 else 0b10000000)):
                             raise ValueError("invalid EIP55 checksum")
-            hash160s.add(cur_hash160)
+            hash160s.add( (cur_hash160, P2PKH_VERSION_BYTE) )

```

Comment:
Why are changes being made to `class WalletEthereum` ?

Reply:
WalletEthereum uses the **_verify_seed** function declared in class WalletBIP32, which I modified in lines ~760-770
Specifically
```
-       if self.pubkey_to_hash160(d_pubkey) in self._known_hash160s:   # expects a list of hashes
+       for hash160, version_byte in self._known_hash160s:             # expects a list of tuples
```